synapse/config/room.py

```diff
@@ -83,4 +96,20 @@ def generate_config_section(self, **kwargs: Any) -> str:
         # will also not affect rooms created by other servers.
         #
         #encryption_enabled_by_default_for_room_type: invite
+
+        # Override the default power levels for rooms created on this server, per
+        # room creation preset.
+        #
+        # Useful if you know that your users need special permissions in rooms
+        # that they create (e.g. to send particular types of state events without
+        # needing an elevated power level).  This takes the same shape as the
+        # `power_level_content_override` parameter in the /createRoom API, but
+        # is applied before that parameter.
+        #
+        # This is something of a workaround in the absence of MSC3779 or MSC3761.
+        #
+        #default_power_level_content_override:
+        #   private_chat: null
```

Comment:
Some example non-null content might help admins understand what goes here:

```suggestion
        #   private_chat: { "events": { "com.example.foo" : 0 } }
```